go/internal/i18n/generate-json.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
avoid python if possible, and especially in the `go/` dir

Reply:
Can I move it in a folder under `/tool/` ?